tests/meta/report/test_meta_presenter.py

```diff
@@ -0,0 +1,52 @@
+"""Tests the Presenter class"""
+from cg.meta.report.presenter import Presenter
+
+
+def test_present_float_string():
+    # GIVEN
+
+    # WHEN formatting None
+    presentable_string = Presenter.process_float_string(None, 0)
+
+    # THEN we should get 'N/A' back
+    assert 'N/A' in presentable_string
```

Comment:
Shouldn't this just be a `==` comparison?

Reply:
done